openruth.module

```diff
@@ -43,6 +43,15 @@ function openruth_ding_provider() {
 }
 
 /**
+ * Implements hook_form_FORM_ID_alter (ding_debts_form)
```

Comment:
Nitpicking here but this should be "hook_form_FORM_ID_alter() for ding_debts_form()" [according to the coding standards](https://www.drupal.org/node/1354#hookimpl).